test/services/cache/ExportItemBuilder.scala

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package services.cache
+
+import java.util.UUID
+
+import forms.declaration.PackageInformation
+
+trait ExportItemBuilder {
+
+  private type ExportItemModifier = ExportItem => ExportItem
+
+  private def generateId(): String = UUID.randomUUID().toString
+
+  def anItem(modifiers: ExportItemModifier*): ExportItem =
+    modifiers.foldLeft(ExportItem(id = generateId())) {
+      case (entry, modifier) => modifier(entry)
+    }
+
+  def withPackageInformations(newInformations: List[PackageInformation]): ExportItemModifier = { item =>
+    item.copy(packageInformation = newInformations)
+  }
+
+  def withPackage(
```

Comment:
We have similar: `ExportsCacheItemBuilder` please check this and use it if it's necessary. If not, tell me why.
As I See withPackage method is similar. The same with whole trait

Reply:
Yep why have we duplicated this?

Reply:
Fixed - i just did not expect that build could name like this

Reply:
Feel free to rename the existing one